web/app/static/js/main.js

```diff
@@ -24,7 +27,10 @@ document.querySelectorAll("#overall-admin-search").forEach((form) =>
 
     await fetch("/search_column", {
       method: "POST",
-      body: JSON.stringify({ col_name: "all" }),
+      body: JSON.stringify({
+        col_name: "all",
+        current_href: window.location.href,
```

Comment:
same here

Reply:
`window.location.href.split('/')[4] ?? ''` should do the trick

Reply:
Unlike python, JS would just return undefined if there will be no 5th element if array, and if so we simply pass empty string, which would be way easier to deal with in python

Reply:
We can't have empty string in this place. I would rather use some comparison like "thing_we_need in body.current_href". Because we need exactly what is inside current page url

Reply:
Will fix it in next PR

Reply:
> We can't have empty string in this place. I would rather use some comparison like "thing_we_need in body.current_href". Because we need exactly what is inside current page url

Yeah, but you can never guarantee this won't be a subject of change in future